charger/openwb/hw/rfid.go

```diff
@@ -0,0 +1,158 @@
+package hw
```

Comment:
...und können wir diesen Ordener noch in native umbenennen analog des TEmplates?

Reply:
Erledigt